tests/swift/test_worker.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+pushd $(dirname "$0")
+DIR=$(pwd)
+RUNBOX="${DIR}/runbox"
+
+echo $RUNBOX
+# Create runbox
+mkdir -p $RUNBOX
+
+# Copy source to runbox
+cp -fv $DIR/script.swift $RUNBOX/script.swift
+cp -fv $DIR/run.stdin $RUNBOX/run.stdin
+
+# Test Compile
+docker run \
+    --cpus="1" \
+    --memory="100m" \
+    --ulimit nofile=64:64 \
+    --rm \
+    --read-only \
+    -v "$RUNBOX":/usr/src/runbox \
+    -v "$RUNBOX":/tmp \
+    -w /usr/src/runbox codingblocks/judge-worker-swift \
+    bash -c "/bin/compile.sh && /bin/run.sh"
+
+ls -lh ${RUNBOX}
+
+expected="Hello World"
+actual="$(cat ${RUNBOX}/run.stdout)"
+if [ "$expected" == "$actual" ] ;then
```

Comment:
how is it coming out to be "Hello World" when according to your source code it should be "HelloWorld"

Reply:
Oddly enough, it works both ways.
`{ print("Hello",input) }`
`{ print("Hello", input) }`
both yield positive tests

Reply:
@championswimmer fixed this, updated `script.swift`